samples/net/ipv4_autoconf/README.rst

```diff
@@ -0,0 +1,25 @@
+.. _ipv4-autoconf-sample:
```

Comment:
Since this is a WIP, I'll assume this placeholder documentation will be expanded on before getting merged.

Reply:
done